.github/workflows/build-container-images.yaml

```
@@ -109,3 +110,13 @@ jobs:
           tags: ghcr.io/rackerlabs/understack/nautobot-interfaces-sync:latest,ghcr.io/rackerlabs/understack/nautobot-interfaces-sync:${{ env.VERSION_PYTHON_NAUTOBOT_INT_SYNC }}
           labels: |
               org.opencontainers.image.version=${{ env.VERSION_PYTHON_NAUTOBOT_INT_SYNC }}
+      - name: Build and deploy Python 3.11 with PyNautobot
+        uses: docker/build-push-action@v5
+        with:
+          context: argo-workflows/ironic-to-nautobot-sync
+          file: argo-workflows/ironic-to-nautobot-sync/containers/Dockerfile
```

```suggestion
          file: containers/Dockerfile
```

Doesn't it need to be that since this is relative to the context? Or wrong?

---

I guess not since it built.